### ISSUE TITLE
Automatic addon submission

### ISSUE THREAD

**Issue by razzeee** [xbmc/metadata.themoviedb.org.python#102]
I see that there have been pipelines https://github.com/xbmc/metadata.themoviedb.org.python/actions/workflows/kodi-addon-submitter.yml but do they work? Do you want help?

There has been a attempt at upstreaming by hand, but automatic would be nicer.

https://github.com/xbmc/repo-scrapers/pull/308

**Comment by razzeee** [comment]
Looking at the pipeline again, it should work fine. So we should try to use it again and maybe check the logs, if it fails.

The only difference to my working example is, that my implementation looks for a tag named `v*` and yours for `number.number.number`

**Comment by razzeee** [comment]
I wasn't aware, that organization can have secrets, but your right, that's the better approach.

**Comment by razzeee** [comment]
No, please don't use the release page after the change. It will be populated automatically, after creating the tag. I don't think github has a nice web frontend for it.

**Comment by rmrector** [comment]
[This release submission](https://github.com/xbmc/repo-scrapers/pull/317) was created with the same janky action as most other version updates. What are you offering to "help"?

To be clear, _you_ ignored the automated process that has always been here and merged a **random GitHub user's changes**, that were not yet merged upstream, into the official add-on repo. For a default add-on bundled with Kodi. This whole process needs drastic change, the action has nothing to do with it.
